templates/enketo/config.json.j2

```diff
@@ -25,9 +25,10 @@
         "email": "{{ enketo_support_email }}"
     },
     "widgets": [
-        "note", "select-desktop", "select-mobile", "autocomplete", "geo", "textarea",
-        "table", "radio", "date", "time", "datetime", "compact", "file", "draw",
-        "likert", "distress", "horizontal", "image-view", "comment", "image-map", "date-mobile"
+        "note", "select-desktop", "select-mobile", "autocomplete", "geo", "textarea", "url",
```

Comment:
@ivermac could we convert this into a variable (I recommend list) with the default values what you now have here?